### ISSUE TITLE
shader not found

### ISSUE THREAD

**Issue by AkinoAki** [SieR-VR/usc-app#10]
i thought this was fixed
![Screenshot_20220903-033520_Unnamed SDVX Clone](https://user-images.githubusercontent.com/91401709/188233394-a0db400e-3402-4b8d-af9e-42651eabc99f.jpg)

Did as told in readme
Run on Galaxy S20U, Android 12
Both on sdcard and internal storage suffer from this error

**Comment by SieR-VR** [comment]
Can you explain your file structure?

**Comment by AkinoAki** [comment]
sdcard/.usc/audio
sdcard/.usc/fonts
sdcard/.usc/skins/Default (this Default folder is the default skin pulled from your link)
Tried my best to follow your guide
Did i miss something?

**Comment by SieR-VR** [comment]
Can you send screenshot in file manager?

**Comment by AkinoAki** [comment]
![Screenshot_20220903-112648_My Files](https://user-images.githubusercontent.com/91401709/188255812-24d8ea0a-60dc-465f-94e0-00df741899c0.jpg)
![Screenshot_20220903-112651_My Files](https://user-images.githubusercontent.com/91401709/188255813-05a18832-8a56-4664-9f2f-5360c86ad0fd.jpg)
![Screenshot_20220903-112656_My Files](https://user-images.githubusercontent.com/91401709/188255817-ed0b7da9-f2fa-48e2-86d6-90cbb0a9e566.jpg)
![Screenshot_20220903-112700_My Files](https://user-images.githubusercontent.com/91401709/188255818-bf72b3de-8c93-487d-a577-232c483fd51c.jpg)
 
Here

**Comment by SieR-VR** [comment]
Did you allow the file management permission?

**Comment by AkinoAki** [comment]
![Screenshot_20220903-113643_Permission controller](https://user-images.githubusercontent.com/91401709/188256041-27eabe44-073e-4019-bd49-e53a1888aa8d.jpg)
Just double checked
Yes i did

**Comment by SieR-VR** [comment]
hm.. I can't find issue

**Comment by SieR-VR** [comment]
If you can, sending log file with logcat will help.

**Comment by AkinoAki** [comment]
I'm outside right now so i'll try that when i get back home

**Comment by AkinoAki** [comment]
[usc logcat.txt](https://github.com/SieR-VR/usc-app/files/9482240/usc.logcat.txt)
 somehow log show the app work as intended and even attempted to load shader from the right directory but still can't find anything

**Comment by AkinoAki** [comment]
I've just noticed that ScanFiles can't run due to "sdcard/.usc/skins/" not recognized as a folder
Why is this happening?

**Comment by fangxx3863** [comment]
I have same problem

**Comment by fangxx3863** [comment]
I found a solution
I found that the file that logcat looks for doesn't match the actual path
LogCat: `SDVX    : Can't run ScanFiles, "/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/skins/" is not a folder`  
Real Path: `/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/usc/skins/
`

**Comment by fangxx3863** [comment]
[Error file](https://github.com/SieR-VR/usc-app/blob/master/app/src/main/assets/usc.zip)  
![image](https://user-images.githubusercontent.com/48589001/194319726-3d389e70-1883-479e-9011-e5f6e17ad0a7.png)

**Comment by hlcm0** [comment]
> I found a solution I found that the file that logcat looks for doesn't match the actual path LogCat: `SDVX : Can't run ScanFiles, "/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/skins/" is not a folder` Real Path: `/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/usc/skins/ `

thanks, helped to fix mine!

**Comment by HiLordReilo** [comment]
> I found a solution
> I found that the file that logcat looks for doesn't match the actual path
> LogCat: `SDVX    : Can't run ScanFiles, "/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/skins/" is not a folder`  
> Real Path: `/storage/emulated/0/Android/data/com.example.unnamed_sdvx_clone/files/usc/skins/
> `

Yeah, this helped me too, but now I had to ditch nautica, multiplayer and challenge courses as the skin itself cannot find "json" module, even though it's present in skin folder. I also tried copying `json.lua` into the same directory as the script that requests it, but it didn't help. I also had to edit all (necessary) lines that load and plays samples, because for some reason they request samples in `"sample"` format while the correct format is `"sample.extension"` (ex: "applause" > "applause.wav")

**Comment by ogami1972** [comment]
same issue. I tried just copying the files to the correct path but that didn't seem to help. Would love to see this on my GPD XD+.
